BizHawk.Client.EmuHawk/tools/Lua/Libraries/EmuLuaLibrary.Client.cs

```diff
@@ -391,6 +391,13 @@ public static int Ypos()
 			return GlobalWin.MainForm.DesktopLocation.Y;
 		}
 
+        [LuaMethodExample("local incbhver = client.getversion( );")]
+        [LuaMethod("getversion", "Returns the current (non-dev) BizHawk version")]
+        public static string GetVersion()
+        {
+            return VersionInfo.GetEmuVersionNonDev();            

```

Comment:
Mainversion is public, why not obtain it directly?

Reply:
I was thinking about appending or prepending something to the string to denote whether it was a release or development build, but then didn't. Can't make up my mind whether this is something that would be useful? If not, we can return directly.

Reply:
If we want to provide interim version (somehow) through lua, it could also obtain the interim version directly I think.